### PR TITLE
Fixing np.NaN error and build warnings

### PR DESCRIPTION
**Pull request by ianhelle**
Fixing pandas deprecation warnings in vtfile_behaviour and process_tree_utils.
The rest is mostly suppressing expected warnings in unit tests

### CODE REVIEW

File: tests/context/test_vtlookupv3.py [hunk 1]
```diff
@@ -161,22 +160,6 @@ def close(self):
         return None
 
 
-def test_init_vt_lookup_class():
```
Comment (ryan-detect-dot-dev) [inline tests/context/test_vtlookupv3.py:164]: Was this deleted on purpose?

**Review by ryan-detect-dot-dev — APPROVED** [review]
LGTM

File: tests/context/test_vtlookupv3.py (hunk 1 above)
Comment (ianhelle) [reply tests/context/test_vtlookupv3.py:164]: Yes it was. It was a trivial test and needed some nasty fix to instantiate VTLookup in an async method. 
I didn't think the effort to change the test was worth the value that the test brought.

**Comment by FlorianBracq** [comment]
That looks good to me too!

**Pull request merged**